src/meta/src/hummock/compaction/picker/intra_compaction_picker.rs

```diff
@@ -243,6 +243,10 @@ impl IntraCompactionPicker {
                 continue;
             }
 
+            if level_handlers[0].is_level_pending_compact(level) {
+                continue;
+            }
```

Comment:
Is this check sufficient? Is it possible that there is a race between a trivial move task and a regular task both outputting to a target sub-level with no SSTs?

Reply:
I do not understand, why there would be race?
If some regular task try to compact target level x from y, the trivial move task to sub level either x or y would be prevent.

Reply:
> I do not understand, why there would be race? If some regular task try to compact target level x from y, the trivial move task to sub level either x or y would be prevent.

Is it possible to has a regular task trying to compact level x, x+1, x+2, x+3 to y and the trivial move task trying to compact task from x+1 to x+2 while x+2 is empty?